packages/manager/modules/emailpro/src/domain/dkim-autoconfig/emailpro-domain-dkim-autoconfig.controller.js

```diff
@@ -144,4 +186,23 @@ export default class EmailProDomainDkimAutoconfigCtrl extends DkimAutoConfigurat
         this.services.$scope.resetAction();
       });
   }
+
+  getBodyText(status) {
+    switch (status) {
+      case this.GLOBAL_DKIM_STATUS.OK:
+        return 'emailpro_tab_domain_diagnostic_dkim_deactivation';
+      case this.GLOBAL_DKIM_STATUS.DISABLED:
+        return 'emailpro_tab_domain_diagnostic_dkim_activation';
+      case this.GLOBAL_DKIM_STATUS.NOT_CONFIGURED:
+        return this.domainDiag.isOvhDomain
+          ? 'emailpro_tab_domain_diagnostic_dkim_activation_ovhcloud'
+          : 'emailpro_tab_domain_diagnostic_dkim_activation_no_ovhcloud';
+      case this.GLOBAL_DKIM_STATUS.NOK:
+        return 'emailpro_tab_domain_diagnostic_dkim_configuration_nok';
+      case this.GLOBAL_DKIM_STATUS.IN_PROGRESS:
+        return '';
+      default:
+        return '';
+    }
```

Comment:
Instead of using a function, you should consider using an object constant (see https://github.com/ovh/manager/pull/10183/files#diff-5d9ae4ea3f3f8fe8830a0472d92b35001c637c8c30e3e57a44c0203e910b9d25).
Moreover, translation should be done here to avoid translating empty value in HTML file -> `data-ng-translate=''`